assets/core/scss/_core.scss

```diff
@@ -305,14 +304,17 @@ textarea {
 */
 
 #main-area {
-  display: table-cell;
```

Comment:
Sorry to bother you, but when was the last time you had problems due to flexbox?